src/app/screens/Keysend/index.tsx

```diff
@@ -110,29 +109,28 @@ function Keysend() {
           <form onSubmit={handleSubmit} className="h-full">
             <Container justifyBetween maxWidth="sm">
               <div>
-                {destination && <PublisherCard title={destination} />}
                 <ContentMessage
                   heading={t("receiver.label")}
                   content={destination}
                 />
-                <div className="p-4 shadow bg-white dark:bg-surface-02dp rounded-lg overflow-hidden">
-                  <DualCurrencyField
-                    id="amount"
-                    label={t("amount.label")}
-                    min={1}
-                    onChange={(e) => setAmountSat(e.target.value)}
-                    value={amountSat}
-                    fiatValue={fiatAmount}
-                  />
-                  <SatButtons onClick={setAmountSat} />
-                </div>
+                <DualCurrencyField
+                  id="amount"
+                  label={t("amount.label")}
+                  min={1}
+                  onChange={(e) => setAmountSat(e.target.value)}
+                  value={amountSat}
+                  fiatValue={fiatAmount}
+                />
+                <SatButtons onClick={setAmountSat} />
```

Comment:
Is SatsButtons needed? doesn't the amount need to be fixed for keysend?

But it doesn't seem to show anyway

![image](https://github.com/getAlby/lightning-browser-extension/assets/33993199/37286859-3ea6-4119-8494-a5f4dbea1db0)

Reply:
I think you are on the wrong screen. :thinking: 

You would have to paste a nodes public key on the send screen to get to the this screen.